bin/create_fortran_test.py

```diff
@@ -119,7 +119,7 @@ def create_test(test_name, json_name):
     j = None
     with open(json_name) as f:
         j = json.load(f)
-    exercise_version = j['version']
+    exercise_version = j['version'] if 'version' in j else None
```

Comment:
The JSON files no longer have versions (see https://github.com/exercism/problem-specifications/issues/1674), so this will always be `None`.

Reply:
Thanks, removed version